internal/pkg/configloader/validation.go

```diff
@@ -85,14 +85,14 @@ func validateConfigFile(c *ConfigFile) []error {
 
 	if c.State == nil {
```

Comment:
Don't change it, let community contributor do it later.

https://github.com/devstream-io/devstream/issues/466